src/editor/codeeditor.cpp

```diff
@@ -9,15 +9,19 @@
 #include<QTextCursor>
```

Comment:
Are all this lexer changes safe to be injected into deveop?

Reply:
Yes, they are) but it's Sasha's responsibility